### ISSUE TITLE
No aria-activedescendant on input

### ISSUE THREAD

**Issue by leo-petrucci** [pacocoursey/cmdk#142]
I'm using `cmdk` within a Radix Popover. Looks just like the example:

https://codesandbox.io/p/sandbox/radix-dropdown-combobox-forked-j8jjph?file=%2Fsrc%2FApp.tsx%3A22%2C49

For some reason, my input doesn't seem to have an `aria-activedescendant` value on it, so when I use the arrow keys up and down to select different values it doesn't read them out to me. 

Is this a feature we should implement ourselves? The examples on [the site](https://cmdk.paco.me/) have the attribute so I assumed it was built in.

Is this something that broke at some point?

**Comment by joaom00** [comment]
Hey @leo-petrucci,

That was fixed in #108 and will be in the next release!
